monitoring/v1/monitoring.proto

```diff
@@ -51,6 +51,71 @@ service MonitoringService {
             body: "*"
         };
     }
+
+    // Get the memory usage metrics for the deployment based on the given request.
+    // Required permissions:
+    // - monitoring.metrics.get on the deployment identified by the given deployment ID.
+    rpc GetDeploymentMemoryUsageMetrics(GetDeploymentMetricsRequest) returns (DeploymentMetrics) {
+        option (google.api.http) = {
+            get: "/api/monitoring/v1/metrics/memory"
+        };
+    }
+
+    // Get the CPU usage metrics for the deployment based on the given request.
+    // Required permissions:
+    // - monitoring.metrics.get on the deployment identified by the given deployment ID.
+    rpc GetDeploymentCPUUsageMetrics(GetDeploymentMetricsRequest) returns (DeploymentMetrics) {
+        option (google.api.http) = {
+            get: "/api/monitoring/v1/metrics/cpu"
+        };
+    }
+
+    // Get the disk usage metrics for the deployment based on the given request.
+    // Note: The response will contain an empty list for coordinators.
+    // Required permissions:
+    // - monitoring.metrics.get on the deployment identified by the given deployment ID.
+    rpc GetDeploymentDiskUsageMetrics(GetDeploymentMetricsRequest) returns (DeploymentMetrics) {
+        option (google.api.http) = {
+            get: "/api/monitoring/v1/metrics/disk"
+        };
+    }
+}
+
+// Request for getting deployment metrics
+message GetDeploymentMetricsRequest {
+    // ID of the deployment for which metrics are being requested.
+    // This is a required field.
+    string deployment_id = 1;
+    // Start time for the query.
+    // This is a required field.
+    google.protobuf.Timestamp start_at = 2;
+    // The end time for the query.
+    // This is a required field.
+    google.protobuf.Timestamp end_at = 3;
+    // Granularity defines the interval between each metric sample.
+    // This needs to be a valid duration.
+    // Example: 1m, 2m, 30s, 1d, etc.
+    // Defaults to 30s.
+    // This is an optional field.
+    string granularity = 4;
+}
+
+
+// DeploymentMetrics contains the deployment metrics
+message DeploymentMetrics {
+    // Sample defines a single data point.
```

Comment:
Note that we do have (potentially) more DBServer / Coordinators imo it would be nice if we render multiple lines (like you see in the sample).
How is this working with the API?

Reply:
<img width="188" alt="image" src="https://github.com/arangodb-managed/apis/assets/47088745/b6873896-992d-4c49-a9a0-4cfcc699f71b">

Reply:
updated